Zend/zend_interfaces.c

```diff
@@ -532,6 +533,13 @@ static int zend_implement_countable(zend_class_entry *interface, zend_class_entr
 }
 /* }}}*/
 
+/* {{{ zend_implement_stringable */
+static int zend_implement_stringable(zend_class_entry *interface, zend_class_entry *class_type)
+{
+	return SUCCESS;
+}
+/* }}}*/
```

Comment:
This is unnecessary.

Reply:
Are you sure? All other functions have it.

Reply:
It's only needed if it actually does something ;)

Reply:
countable has the same, why so?
removed!